### PR TITLE
Add multiple epoch rewards claimer, use kyber utils sc

### PR DESCRIPTION
**Pull request by Anyhowclick**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
   "author": "Kyber SC Team",
   "license": "MIT",
   "dependencies": {
+    "@kyber.network/utils-sc": "^2.0.0",
```
Comment (ilanDoron) [inline package.json:24]: I think we should fix this.
but later it should be 2.1.0 after Anton is done.

**Review by ilanDoron — APPROVED** [review]
can probably PR to master
right?

File: contracts/IERC20.sol [hunk 2]
```diff
@@ -1,30 +1,3 @@
...
-
-    function totalSupply() external view returns (uint256 supply);
-}
-
-
-// to support backward compatible contract name -- so function signature remains same
-abstract contract ERC20 is IERC20 {
-
-}
+import "@kyber.network/utils-sc/IERC20.sol";
```
Comment (ilanDoron) [inline contracts/IERC20.sol:3]: not sure what is the advantage for using the imported one?

**Comment by Anyhowclick** [comment]
Develop branch and master branch not in sync, not sure which one was to be used.

File: contracts/IERC20.sol (hunk 2 above)
Comment (Anyhowclick) [reply contracts/IERC20.sol:3]: Not an advantage. I run into compilation issues if I don't use the imported one. Mainly how the IERC20 contract has already been defined.

File: package.json (hunk 1 above)
Comment (Anyhowclick) [reply package.json:24]: Yes, will wait for Anton to be done.

**Review by ilanDoron — COMMENTED** [review]
please move this contract to wrapper folder.
and also the interface.

I think we don't need it in main folder.

**Review by ilanDoron — APPROVED** [review]

**Review by ilanDoron — APPROVED** [review]

**Pull request closed without merging**